webapp/src/Components/Pages/DocumentationPage.tsx

```diff
@@ -1,16 +1,86 @@
-import { Card, CardHeader, Divider, Grid, Table, TableBody, TableCell, TableRow } from "@material-ui/core"
+import {
+    Card,
+    CardHeader,
+    Divider,
+    Grid,
+    Table,
+    TableBody,
+    TableCell,
+    TableRow,
+} from "@material-ui/core"
 import * as React from "react"
 import { getDocumentation } from "../../Requests/Documentation"
 import { LoadableWrapper } from "../Shared/LoadableWrapper"
 import { BasePage } from "./BasePage"
-import ReactJson from "react-json-view";
-
+import ReactJson from "react-json-view"
 
 interface State {
     reloadSignal: boolean
     explanations?: Record<string, any>
 }
 
+interface Props {
+    queryParam: any
+}
+
+class QueryParams  extends React.PureComponent<Props> {
+    public render() {
+        return (
+            <>
+                <TableRow>
```

Comment:
If you're going to have only one top-level component, the fragment is redundant in line 29